site/specs-temp/numbers.json

```diff
@@ -13352,6 +13352,103 @@
         }
       }
     },
+    "/accounts/{accountId}/shortcodes": {
+      "get": {
+        "tags": [
+          "Shortcodes"
+        ],
+        "description": "Retrieve shortcodes",
+        "summary": "Retrieve shortcodes",
+        "operationId": "listShortcodes",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Retrieve list shortcodes",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/shortCodeResponseList"
+                },
+                "examples": {
+                  "example": {
+                    "value": "<ShortCodeResponseList>\n  <ShortCode>\n    <ShortCodeNumber>37220</ShortCodeNumber>\n    <AccoundId>9901078</AccoundId>\n    <SipPeerId>23234</SipPeerId>\n    <Country>USA</Country>\n    <DateFirstOrdered>2021-06-27T23:31:06Z</DateFirstOrdered>\n    <LeaseStartDate>2022-06-27T23:31:06Z</LeaseStartDate>\n    <LeaseEndDate>2022-12-28T00:31:06Z</LeaseEndDate>\n    <Status>ACTIVE</Status>\n    <AttStatus>INACTIVE</AttStatus>\n    <TmoStatus>ACTIVE</TmoStatus>\n    <VerizonStatus>ACTIVE</VerizonStatus>\n    <Type>VANITY</Type>\n    <CustomerManagedLease>false</CustomerManagedLease>\n  </ShortCode>\n  <ShortCode>\n    <ShortCodeNumber>37220</ShortCodeNumber>\n    <AccoundId>9901078</AccoundId>\n    <SipPeerId>23234</SipPeerId>\n    <Country>CAN</Country>\n    <DateFirstOrdered>2021-06-27T23:31:06Z</DateFirstOrdered>\n    <LeaseStartDate>2022-06-27T23:31:06Z</LeaseStartDate>\n    <LeaseEndDate>2022-12-28T00:31:06Z</LeaseEndDate>\n    <Status>ACTIVE</Status>\n    <AttStatus>INACTIVE</AttStatus>\n    <TmoStatus>ACTIVE</TmoStatus>\n    <VerizonStatus>ACTIVE</VerizonStatus>\n    <Type>VANITY</Type>\n    <CustomerManagedLease>false</CustomerManagedLease>\n  </ShortCode>\n</ShortCodeResponseList>"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/shortcodes/{shortcode}": {
+      "get": {
+        "tags": [
+          "Shortcodes"
+        ],
+        "description": "Retrieve shortcode",
+        "summary": "Retrieve shortcode",
+        "operationId": "getShortcode",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Include 202 response in case provisioning isn't completed.

Reply:
Need a description for 202.